### PR TITLE
Add notes for CG-05-23.md

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

**Review by conrad-watt — APPROVED** [review]
Oh, thanks! I should remember to handle this in future.

File: main/2023/CG-05-23.md [hunk 1]
```diff
@@ -28,12 +28,227 @@ Installation is required, see the calendar invite.
...
+FM: you’ll replace the existing APIs in Java with those provided by stringref
+
+ZB: You can use basic things and implement other APIs on top.
+
+AK: for example locale toLower, thats not in stringref, it’s in the intl library in JS. I would expect stringref to be low-level enough that issues about library support would be out of scope.
+
+CW: Even in linear memory, you would have the same overhead of copying it in and copying out back? 
+
+AR: I don’t buy that it’s as simple as that. There's a very strong slippery slope. This is already pulling some amount of unicode logic into the engine/platform and people seem to be already requesting more. Si this is a very real risk that we do end up putting things from Java into the platform. These arguments are a fallacy because you could use them to argue for integrating any feature from JS or another language. So if you want to integrate with that then of course youd benefit from adding thnose elements. But this is 
+The opposite of what we want ot do for wasm, where we want to add as little on top of the hardware as we can get away with. It has big implications beyond looking at this feature.
```
Comment (tlively) [inline main/2023/CG-05-23.md:145]:
```suggestion
The opposite of what we want to do for wasm, where we want to add as little on top of the hardware as we can get away with. It has big implications beyond looking at this feature.
```

File: main/2023/CG-05-23.md [hunk 2]
```diff
@@ -28,12 +28,227 @@ Installation is required, see the calendar invite.
...
+
+AVK (via chat): Yeah, that's the "Latin1" strings I asked. about.
+
+AK presenting import-based strings
+
+AR: it’s worth pointing out that type imports are not strictly necessary. You can get pretty far with just externref. I would be interested in seeing how far you get, especially given all the other stuff JS engines do.
+
+CW: Would this still be externref with pre-imports though?
+
+AR: If you don’t use pre-imports, I would externref to be more expensive with all the depots,
```
Comment (tlively) [inline main/2023/CG-05-23.md:179]:
```suggestion
AR: If you don’t use pre-imports, I would expect externref to be more expensive with all the deopts,
```

**Comment by dschuff** [comment]
Thanks Thomas.
Conrad, no worries, I just noticed that I still had a tab open with the doc so I moved it over.

**Pull request merged**